vowpalwabbit/cbify.cc

```diff
@@ -75,11 +51,6 @@ template<class T> inline void delete_it(T* p) { if (p != nullptr) delete p; }
 void finish(cbify& data)
 {
   CB::cb_label.delete_label(&data.cb_label);
-  //data.probs.delete_v();
-  delete_it(data.scorer);
-  delete_it(data.generic_explorer);
-  delete_it(data.mwt_explorer);
-  delete_it(data.recorder);
```

Comment:
The deleting of all this stuff is great.  It says we have a better interface now.